### ISSUE TITLE
v2.8.7: no docker image for hybrid? 

### ISSUE THREAD

**Issue by bitsofinfo** [TykTechnologies/tyk#3405]
another image that is missing at dockerhub v2.8.7 hybrid

please update all the missing images. trying to upgrade and this is becoming a pain of trial error of which version is published/which is not.

**Comment by alephnull** [comment]
Looking for missing tags after v2.8

```shellsession
% for v in $(git tag | grep "^v[23].[890]" | grep -v -- "-rc")
do
docker manifest inspect tykio/tyk-hybrid-docker:${v} >/dev/null 2>&1 || print $v
done
v2.8.6
v2.8.7
v2.9.3.2
v2.9.4
v2.9.4.6
v3.0.1-ara.2
```

The v2.8.7 build failed due to missing packages in packagecloud (or perhaps tagged with the wrong version).

**Comment by excieve** [comment]
@alephnull The build fails because the `Dockerfile` is a 2.9+ template rather then the old style (with jessie and old Python).

**Comment by alephnull** [comment]
v2.8.7 pushed by hand to Hub and the fixed Dockerfile committed to release-2.8 branch on tyk-hybrid-docker.

**Comment by bitsofinfo** [comment]
so this is now pushed ok? can this be closed?
